content/docs/components-and-props.md

```diff
@@ -118,15 +127,16 @@ ReactDOM.render(
 );
 ```
 
-[](codepen://components-and-props/composing-components)
+[Probeer het op CodePen](codepen://components-and-props/composing-components)
 
-Typically, new React apps have a single `App` component at the very top. However, if you integrate React into an existing app, you might start bottom-up with a small component like `Button` and gradually work your way to the top of the view hierarchy.
+Normaalgesproken hebben React applicaties helemaal bovenin een enkele `App` component.
+Als je React echter integreert in een bestaande applicatie, kan het zijn dat je van onderaf begint met een kleine component, zoals een `Button`, en dan langzaam naar boven werkt in de view hierarchie.
 
-## Extracting Components {#extracting-components}
+## Components extraheren {#extracting-components}
```

Comment:
```suggestion
## Components Opsplitsen {#extracting-components}
```
I personally prefer this. In any case, there's a missing capital letter in `extraheren`.

Reply:
I couldn't come up with a better translation, but "Opsplitsen" is definitely better and easier.